conf/multiconfig/rbuilder.conf

```diff
@@ -0,0 +1,6 @@
+# meta-evm deps
+PACKAGE_INSTALL:append:pn-core-image-tiny-initramfs = " rbuilder orderflow-proxy system-api ssh-key-injection"
+# meta-custom-podman deps
+PACKAGE_INSTALL:append:pn-core-image-tiny-initramfs = " podman"
+# meta- deps
+PACKAGE_INSTALL:append:pn-core-image-tiny-initramfs = " fluent-bit prometheus node-exporter process-exporter mustache"
```

Comment:
How are those different than the code inside [core-image-tiny-initramfs.inc](https://github.com/flashbots/meta-evm/blob/a270b66238de16ff57870a868db232dd2d0cf547/recipes-core/images/core-image-tiny-initramfs.inc)?

These are adding the dependencies directly here and there too?
Aren't those redundant? because the `core-image-tiny-initramfs.inc` is already including those recipes needed for the image and the recipes are having the other required dependencies in their RDEPENDS

Reply:
I've changed this to appending the distro features for consistency